pages/index.js

```diff
@@ -21,28 +21,27 @@ export default function Index() {
             <div>
               <h2>Local-first</h2>
               <p>
-                All data, including issues, comments and other social artifacts is yours & lives on your machine.
+                All data, including issues, comments and reviews, is yours & lives on your machine. Everything is always available - even offline.
               </p>
             </div>
             <div>
-              <h2>Secure &amp; Trustless</h2>
+              <h2>Secure &amp; trustless</h2>
               <p>
-                Cryptographic peer identities and way better keys that take security to another level.
-                Shared ownership of projects & everything that comes with that.
+                All your artifacts (yes, even reviews) are cryptographically signed and backed by secure peer identities so you don't have to trust intermediaries to collaborate and verify.
               </p>
             </div>
             <div>
-              <h2>Sustainable</h2>
+              <h2>Sustainable &amp; community-owned </h2>
               <p>
-                Enabling developers to monetize their work while releasing their software in the open.
+                Support and fund your open source projects, contributions, and dependencies with wallets and shared multi-sigs in a network collectively owned by the FOSS community.
```

Comment:
I think multi-sig is more a part of trustless.

Reply:
Okay. Do we like the community powered part of this?

Reply:
Yeah, I think it's interesting.